File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(),g.end());
        sort(s.begin(),s.end());
        int i=0;
        int j=0;
        int size1 = g.size();
        int count =0;
        int size2 = s.size();
        while(i<size1 && j<size2){
            if(g[i]<=s[j]){
                i++;
                j++;
                count++;
                
            }
            else j++;
        }
        return count;
        
    }
};

File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        i=0
        j=0
        n = len(g)
        m = len(s)
        ans = 0
        while i<n and j<m:
            if s[j] >= g[i]:
                ans = ans + 1
                j = j+1
                i = i+1
            else:
                j = j + 1
        return ans
        